### PR TITLE
Ref-based RNA-seq: fix DESeq parameter setting and add link to group tags tutorial

### PR DESCRIPTION
**Pull request by shiltemann**
and a fix to the group tags tutorial

### CODE REVIEW

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 1]
```diff
@@ -732,14 +732,15 @@ Here, treatment is the primary factor that we are interested in. The sequencing
...
->    - *"how"*: `Select group tags corresponding to levels`
+>    - *"how"*: `Select datasets per level`
 >       - In *"Factor"*:
 >          - In "1: Factor"
 >              - *"Specify a factor name"*: `Treatment`
 >              - In *"Factor level"*:
 >                  - In *"1: Factor level"*:
 >                      - *"Specify a factor level"*: `treated`
 >                      - {% icon param-files %} *"Counts file(s)"*: the 3 gene count files with `treat` in their name
+>                         - Tip: use the CTRL key to select multiple files
```
Comment (erasche) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md]:
```suggestion
>                         - Tip: use the <kbd>Ctrl</kbd> key to select multiple files
```

Comment (shiltemann) [reply topics/transcriptomics/tutorials/ref-based/tutorial.md]: thanks :)

Comment (bebatut) [reply topics/transcriptomics/tutorials/ref-based/tutorial.md]: you could also add the tip box here for multiple dataset selection

Comment (shiltemann) [reply topics/transcriptomics/tutorials/ref-based/tutorial.md]: oh, good idea, didn't know we had a snippet for that :)

**Pull request merged**